lib/splash.js

```diff
@@ -39,7 +41,13 @@ function logKey(key) {
 }
 
 function logValue(key, value) {
-    return "| " + indent() + " " + key + ": " + value + "\n";
+    var hideValue = _.reduce(KEYS_TO_HIDE, function(memo, keyToHide) {
+        return memo || key.indexOf(keyToHide) >= 0;
```

Comment:
I get the following error when the configuration property is an array:
```
/home/sebz/dev/swir/av/node-tech-logger/lib/splash.js:46
        return memo || key.indexOf(keyToHide) >= 0;
                           ^
TypeError: Object 0 has no method 'indexOf'
    at /home/sebz/dev/swir/av/node-tech-logger/lib/splash.js:46:28
```

In `av-ui` we have:
```yaml
# Supported locales
locales : ["en", "fr"]
```

We may need a new function `logArray`..?